cou/logging.py

```diff
@@ -76,7 +78,8 @@ def setup_logging(log_level: str = "INFO") -> None:
 
     root_logger.addHandler(log_file_handler)
     root_logger.addHandler(console_handler)
-    logger.info("Logs of this execution can be found at %s", file_name)
+
+    progress_indicator.stop_and_persist(text=f"Logs of this execution can be found at {file_name}")
```

Comment:
I don't have the context of cou so I feel wried that why we need to setup progress_indicator in a "setup_logging" function.  Maybe we should decouple it?

Reply:
You will see
```bash
$ cou run
Configuring logging... /  # the / will rotate to show progress
```
after that you will see
```bash
$ cou run
Logs of this execution can be found at <file-name>.log ✔
Connected to 'test-model' ✔
Analyzing cloud... ✔
Generating upgrade plan... ✔
...
```

Reply:
BTW, Please feel free to resolve the conversation. I just want to ask some questions but not blocking the merge.

Reply:
I think I do not understand, the indicator is not sent to anywhere. Let my try to explain how it's works, maybe it will be helpful.
We are creating indicator objects in utils, with some configuration of our choice.  This indicator object is following singleton pattern, so we can use the same (per each run) indicator everywhere.
I'm glad that you are interested, so happy to provide any answer.